### PR TITLE
add extra view for details

### PR DESCRIPTION
**Pull request by 2pd**
image

### CODE REVIEW

**Comment by 2pd** [comment]
What do you think if we add an extra view for details, it's convenience to put more info later

File: main.go [hunk 1]
```diff
@@ -208,10 +212,23 @@ func layout(g *gocui.Gui) error {
...
+
+	// show some details
+	if v, err := g.SetView("details", int(0.2*float32(maxX)), int(0.8*float32(maxY)), maxX, maxY); err != nil {
+
+		if err != gocui.ErrUnknownView {
+			return err
+		}
+
+		v.Title = "Details"
+		v.Highlight = false
```
Comment (y-yagi) [inline main.go:233]: Please specify `v.Wrap = true` for long sentence.

File: main.go [hunk 2]
```diff
@@ -346,6 +396,20 @@ func cursorRight(g *gocui.Gui, v *gocui.View) error {
...
+func refreshDetailsView(g *gocui.Gui) error {
+	mainView, _ := g.View("main")
+	_, cy := mainView.Cursor()
+
+	detailsView, _ := g.View("details")
+	detailsView.Clear()
+
+	repo := reposPerLang[lang][cy]
+
+	fmt.Fprintf(detailsView, "%s %s\n", repo.language, repo.stars)
```
Comment (y-yagi) [inline main.go]: How do you feel add `☆` before show stars?
(I think it's probably a problem to get used to, but I didn't know any numbers at first.)

**Review by y-yagi — COMMENTED** [review]
I just added a minor comment. But seems good :+1:

**Comment by 2pd** [comment]
😄

**Comment by y-yagi** [comment]
:clap: :clap: :clap:

**Pull request merged**